src/manage-apps/manage-repos-repoman.md

```diff
@@ -0,0 +1,92 @@
+# Managing Repositories with Repoman
+
+Add, remove, and edit repositories using Repoman. Repoman allows users to add apt and Flatpak repositories. Users may want to add an apt repository or additional Flatpak repository if their software is not included in the Pop!\_Shop, or they may want to remove an unneeded repository after uninstalling the only application that required it.
+
+Access Repoman by launching the Pop!\_Shop, clicking the three lines, and then clicking `System Software Sources...`.
+
+![Access Repoman](/images/manage-repos/access-repoman.png)
+
+>**Caution**: Adding 3rd-party apt or Flatpak repositories allows installation of software that has not been validated by System76 or other trusted Linux communities. Software available via 3rd-party apt and Flatpak repositories may not be vetted against packages that introduce security vulnerabilities. Users should take caution and only add external repositories that they trust.
+---
+
+## Official Sources
+
+This section lists System76's official Pop!\_OS mirror. This mirror includes installation files and updates for all applications included in a default Pop!\_OS installation. This is also where installation files for many applications in the Pop!\_Shop reside. Changing this mirror is not necessary since Pop!\_OS will automatically download updates from the fastest source using System76's worldwide CDN.
```

Comment:
> This section lists System76's official Pop!\_OS mirror.

To be more accurate, this section lists our Ubuntu mirror.

> This mirror includes installation files and updates for all applications included in a default Pop!\_OS installation.

That is false. It only hosts packages that come from Ubuntu. Packages that we add on top to make Pop!_OS come from the `apt.pop-os.org/release` repo, which is on the `Extra Sources` page. (Seeing this documented has made me start to wonder if the UX should be different for this.)

Reply:
Corrected this and added info about the /release repo.